flask_session/sessions.py

```diff
@@ -131,11 +131,15 @@ def open_session(self, app, request):
             sid = sid.decode('utf-8', 'strict')
         val = self.redis.get(self.key_prefix + sid)
         if val is not None:
+            if '/user/login' in request.path:
```

Comment:
There's no guarantee whatsoever that this URL is used to log in!

I believe for a library that has no knowledge about the contents of the session, it is impossible to prevent session fixation. All you can do there is providing provide an interface to regenerate the session ID which the application then calls when logging in/out a user.